.devcontainer/Dockerfile

```diff
@@ -17,7 +17,7 @@ COPY ./ ./
 # setup.py uses a task from Makefile.devcontainer but expects it to be in the default Makefile
 COPY ./Makefile.devcontainer ./Makefile
 
-RUN --mount=type=cache,target=/var/cache/dnf && \
+RUN --mount=type=cache,target=/var/cache/dnf \
```

Comment:
does this mean the previous command wasn't working though?

Reply:
Yes, it wasn't unfortunately :|